circle.yml

```diff
@@ -24,7 +24,7 @@ general:
 
 deployment:
   release:
-    tag: /(([a-z]+)-)?([0-9]+)\.([0-9]+)\.([0-9]+)/
+    tag: /(([a-z]+)-)*([0-9]+)\.([0-9]+)\.([0-9]+)/
```

Comment:
> What does the script produce when the `CIRCLE_TAG` variable is set to an umbrella tag?

From [referred to comment][1]

```
$ CIRCLE_TAG=0.22.0 python scripts/circleci_tagged_pkg.py
/home/dhermes/da_googz/worktree_regex/
```

In [the source][2] it just returns the `git` root without trying to append a subdir

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2853#issuecomment-266242369
[2]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/b004bd43ecc0d2eb80367afc9615e135a95726d5/scripts/circleci_tagged_pkg.py#L52-L53